### DISCUSSION TITLE
blog/mindful-commit-message

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#168] [General]
# blog/mindful-commit-message

These guides and set of rules can help us to understand what is going on just by looking at the git log.

https://theodorusclarence.com/blog/mindful-commit-message

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Comment by ryan-zayne** [comment]
It's honestly been a blessing that I managed to stumble across this blog🔥🔥
